huaweicloud/services/dew/data_source_huaweicloud_csms_secret_version.go

```diff
@@ -0,0 +1,94 @@
+package dew
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/chnsz/golangsdk/openstack/csms/v1/secrets"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+)
+
+func DataSourceDewCsmsSecret() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceDewCsmsSecretRead,
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"secret_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"version": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"secret_text": {
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
```

Comment:
just a question, if we set Sensitive here, can we reference it directly?

Reply:
yes, I have verified it